### DISCUSSION TITLE
how is redbean-unsecure built?

### DISCUSSION THREAD

**Discussion by xfnw** [jart/cosmopolitan#390] [Q&A] [answered]
[the website](https://redbean.dev/#source) shows how to build full redbean including ssl, but not how to build redbean-unsecure (without ssl)

**Comment by pkulchenko** [comment, marked as answer]
@xfnw, using `make -j8 o//tool/net/redbean-unsecure.com` should build it. In general, everything you see as a target in tool/net/net.mk (for example, redbean-static.com, redbean-demo.com, and redbean-original.com) can be specified and built this way.
